test/e2e/fixture-builder.js

```diff
@@ -511,6 +511,29 @@ class FixtureBuilder {
     return this;
   }
 
+  withNetworkControllerDoubleGanache() {
+    return this.withNetworkController({
+      networkConfigurations: {
+        networkConfigurationId: {
+          chainId: CHAIN_IDS.LOCALHOST,
+          nickname: 'Localhost 8545',
+          rpcPrefs: {},
+          rpcUrl: 'http://localhost:8545',
+          ticker: 'ETH',
+          networkConfigurationId: 'networkConfigurationId',
+        },
+        '76e9cd59-d8e2-47e7-b369-9c205ccb602c': {
+          id: '76e9cd59-d8e2-47e7-b369-9c205ccb602c',
```

Comment:
`networkConfigurationId2` or something like that? no strong preference, just wondering if uuid is necessary